x-pack/plugins/cloud_security_posture/public/pages/rules/index.tsx

```diff
@@ -4,28 +4,86 @@
  * 2.0; you may not use this file except in compliance with the Elastic License
  * 2.0.
  */
-import React from 'react';
-import type { EuiPageHeaderProps } from '@elastic/eui';
+import React, { useMemo } from 'react';
+import { useQuery } from 'react-query';
+import { RouteComponentProps } from 'react-router-dom';
+import { EuiText, EuiTextColor, EuiEmptyPrompt } from '@elastic/eui';
+import { FormattedMessage } from '@kbn/i18n-react';
 import { CspPageTemplate } from '../../components/page_template';
-import { RulesContainer } from './rules_container';
+import { RulesContainer, type PageUrlParams } from './rules_container';
 import { allNavigationItems } from '../../common/navigation/constants';
 import { useCspBreadcrumbs } from '../../common/navigation/use_csp_breadcrumbs';
+import { useKibana } from '../../common/hooks/use_kibana';
+import { type PackagePolicy, packagePolicyRouteService } from '../../../../../plugins/fleet/common';
+import type { KibanaPageTemplateProps } from '../../../../../../src/plugins/kibana_react/public';
+import { CspLoadingState } from '../../components/csp_loading_state';
+import { CspNavigationItem } from '../../common/navigation/types';
 
-// TODO:
-// - get selected integration
-
-const pageHeader: EuiPageHeaderProps = {
-  pageTitle: 'Rules',
+const useCspIntegrationInfo = ({ packagePolicyId }: PageUrlParams) => {
+  const { http } = useKibana().services;
+  return useQuery(
+    ['packagePolicy', { packagePolicyId }],
+    () => http.get<{ item: PackagePolicy }>(packagePolicyRouteService.getInfoPath(packagePolicyId)),
+    { select: (response) => response.item, enabled: !!packagePolicyId }
+  );
 };
 
-const breadcrumbs = [allNavigationItems.rules];
+const getRulesBreadcrumbs = (name?: string): CspNavigationItem[] =>
+  [allNavigationItems.benchmarks, { ...allNavigationItems.rules, name }].filter(
+    (breadcrumb): breadcrumb is CspNavigationItem => !!breadcrumb.name
+  );
+
+export const Rules = ({ match: { params } }: RouteComponentProps<PageUrlParams>) => {
+  const integrationInfo = useCspIntegrationInfo(params);
+  const breadcrumbs = useMemo(
+    // TODO: make benchmark breadcrumb navigable
+    () => getRulesBreadcrumbs(integrationInfo.data?.name),
+    [integrationInfo.data?.name]
```

Comment:
the breadcrumbs add `path` instead of `href` so they're not links, needs to be addressed in a separate PR